### PR TITLE
New camera

### PR DESCRIPTION
**Pull request by dankirsdot**
I added simple camera to my model. You can run via 'gazebo.launch' file in the arm_sensor package. Captured images are stored in 'arm_sensor/scripts/images' folder. I added some of them into the repository to show the example. The video is [here](https://youtu.be/q-WrXrEHGDs)

### CODE REVIEW

**Comment by Terminateit** [comment]
Lol(2)

File: arm_sensor/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED)
+
+## System dependencies are found with CMake's conventions
```
Comment (l1va) [inline arm_sensor/CMakeLists.txt]: are you really need all these comments?

File: arm_sensor/scripts/camera.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+# OpenCV2 for saving an image
+import cv2
+
+# Instantiate CvBridge
+bridge = CvBridge()
+
+
+import os
+path = os.path.dirname(__file__)
+os.chdir(path)
```
Comment (l1va) [inline arm_sensor/scripts/camera.py]: why not just use path - and add to the rest of the path, without changing dir?

Comment (dankirsdot) [reply arm_sensor/scripts/camera.py]: Actually, no. I deleted all unnecessary lines.

File: arm_sensor/CMakeLists.txt (hunk 1 above)
Comment (dankirsdot) [reply arm_sensor/CMakeLists.txt]: Actually, no. I deleted all unnecessary lines.

File: arm_sensor/scripts/camera.py (hunk 2 above)
Comment (dankirsdot) [reply arm_sensor/scripts/camera.py]: Updates the camera.py script. For now on it uses rospgk to obtain the path to the package and creates the folder to store images automatically

**Pull request merged**